boards/amd/acp_6_0_adsp/acp_6_0_adsp_defconfig

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_GEN_ISR_TABLES=y
+CONFIG_GEN_IRQ_VECTOR_TABLE=y
+CONFIG_XTENSA_RESET_VECTOR=y
+CONFIG_OUTPUT_SYMBOLS=y
+CONFIG_MULTI_LEVEL_INTERRUPTS=n
+CONFIG_2ND_LEVEL_INTERRUPTS=n
+CONFIG_BUILD_OUTPUT_BIN=n
```

Comment:
This one should be forced off at the arch layer, no?  If you're getting a zephyr.bin (which is mostly only meaningful on cortex-M) there's a bug somewhere we should fix.   It's not supposed to be something every board needs to turn off.